docs/public/specifications/tests/Address-TP/address01.md

```diff
@@ -1,47 +1,154 @@
+
+This test checks the name server IP addresses, both those derived from delegation and those 
+derived from the name servers listed in the zone (NS records). Each address is compared against
+the IANA databases. If an address is not globally routable, a message is outputted.
+
+## Scope
+
+This test case does not do any  actual connectivity test, which is done by other test cases.
+
+## Inputs
+
+* "Child Zone" -- the domain name to be tested.
+
+## Summary
 
-### Inputs
+Message Tag                       | Level    | Arguments | Message ID for message tag
+:-------------------------------- |:---------|:----------|:--------------------------
+A01_ADDR_GLOBALLY_REACHABLE       | INFO     |           | All IP addresses of all name servers are listed as in the globally reachable address space.
+A01_NO_NAME_SERVERS_FOUND         | CRITICAL |           | No name servers found.
+A01_NO_GLOBALLY_REACHABLE_ADDR    | CRITICAL |           | None of the name servers IP addresses are listed as globally reachable.
+A01_ADDR_NOT_GLOBALLY_REACHABLE   | ERROR    | ns_list   | IP address not listed as globally reachable "{ns_list}".
+A01_DOCUMENTATION_ADDR            | ERROR    | ns_list   | IP address intended for documentation purposes "{ns_list}".
+A01_LOCAL_USE_ADDR                | ERROR    | ns_list   | IP address intended for local use on network or service provider level "{ns_list}". 
 
-The domain name to be tested.
 
-### Ordered description of steps to be taken to execute the test case
+The value in the Level column is the default severity level of the message. The
+severity level can be changed in the [Zonemaster-Engine profile]. Also see the
+[Severity Level Definitions] document.
 
-1. Obtain the IP addresses of each name server of the domain from the parent using
-   [Method4](../Methods.md) and child using [Method5](../Methods.md)
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [Argument list].
 
-2. If any IP address (IPv4 or IPv6) falls within any of the blocks in the below
-   mentioned IANA links, the test case fails:
-   * <https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml>
-   * <https://www.iana.org/assignments/iana-ipv6-special-registry/iana-ipv6-special-registry.xhtml>
+## Test procedure 
 
-### Outcome(s)
+1. Create the following empty sets:
+   1. Name server name and IP address ("Name Server IP").
+   2. Name server name and IP address ("Documentation Address").
+   3. Name server name and IP address ("Local Use Address").
+   4. Name server name and IP address ("Not Globally Reachable").
 
-If one name server has one of its addresses matches a forbidden address
-block, the test fails. If all the name server addresses are outside these
-forbidden blocks, the test case succeeds. 
+2. Obtain the glue address records of each name server for the *Child Zone* from the
```

Comment:
I think "Obtain the address records" is more correct (removing) since the addresses are not necessarily glue records in the correct sence.